hail/python/hail/ir/ir.py

```diff
@@ -569,7 +570,7 @@ def __eq__(self, other):
 
 
 class ArrayMap(IR):
-    @typecheck_method(a=IR, name=str, body=IR)
+    @typecheck_method(a=IR, name=oneof(str, Symbol), body=IR)
```

Comment:
Ah, I can do that here, but not at the higher level of the interface (e.g. tstruct) because if I convert str to Symbol, then iterating over the fields, say, returns Symbols instead of strings, which we don't want.  Then the question is, should we be consistent through the Python code base, and I went for that.  In fact, I think I should just delete Identifier from the Python side.

Objections?

Reply:
oh, yeah, very reasonable.